src/Core/AdminConsole/Services/Implementations/OrganizationService.cs

```diff
@@ -811,6 +815,23 @@ private async Task ValidateSignUpPoliciesAsync(Guid ownerId)
         }
     }
 
+    public async Task InitiateDeleteAsync(Organization organization, string orgAdminEmail)
+    {
+        var orgAdmin = await _userRepository.GetByEmailAsync(orgAdminEmail);
+        if (orgAdmin == null)
+        {
+            throw new BadRequestException("Org admin not found.");
+        }
+        var orgAdminOrgUser = await _organizationUserRepository.GetDetailsByUserAsync(orgAdmin.Id, organization.Id);
+        if (orgAdminOrgUser == null || orgAdminOrgUser.Status != OrganizationUserStatusType.Confirmed ||
+            (orgAdminOrgUser.Type != OrganizationUserType.Admin && orgAdminOrgUser.Type != OrganizationUserType.Owner))
```

Comment:
@kspearrin / @r-tome currently only Owners can delete organizations via the app. Are we intentionally broadening it to Admins here? To be fair I'm not sure what CS' practices are, maybe in an emergency/lockout situation we need to allow for the Owner being unavailable - although that seems like a situation we'd want to be careful about.